api/responses.py

```diff
@@ -0,0 +1,24 @@
+from flask import make_response
```

Comment:
🏕️ This will allow better standardisation of our API, we should think about having a 'funding_utils' git repo we can use as a submodule.

Reply:
Yes good idea